build.gradle

```diff
@@ -11,28 +11,29 @@ buildscript {
     // changing them. Please note that, for using in Android-Components, the
     // versions below must match the ones in that repository.
     ext.versions = [
-        android_components: '5.0.0',
+        android_components: '15.0.0',
```

Comment:
Whoa! That was a big jump.

Reply:
Ehehhe, yeah :) We're just using this for tests, so it should not be too pressing to update it :D